### ISSUE TITLE
`clams source` command not working under 0.4.1

### ISSUE THREAD

**Issue by keighrim** [clamsproject/clams-python#72]

**Comment by kelleyl** [comment]
here is the error I got running clams source with clams-python 0.4.1 

```Traceback (most recent call last):
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/bin/clams", line 11, in <module>
    load_entry_point('clams-python==0.4.1', 'console_scripts', 'clams')()
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/setuptools-39.1.0-py3.8.egg/pkg_resources/__init__.py", line 480, in load_entry_point
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/setuptools-39.1.0-py3.8.egg/pkg_resources/__init__.py", line 2691, in load_entry_point
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/setuptools-39.1.0-py3.8.egg/pkg_resources/__init__.py", line 2322, in load
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/setuptools-39.1.0-py3.8.egg/pkg_resources/__init__.py", line 2328, in resolve
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/clams/__init__.py", line 1, in <module>
    from clams.app import *
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/clams/app/__init__.py", line 12, in <module>
    from clams.appmetadata import AppMetadata
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/clams/appmetadata/__init__.py", line 76, in <module>
    class RuntimeParameter(_BaseModel):
  File "/Users/kelleylynch/PycharmProjects/universal-viewer-consumer/venv/lib/python3.8/site-packages/clams/appmetadata/__init__.py", line 84, in RuntimeParameter
    type: param_value_types = pydantic.Field(..., description=f"Type of the parameter value the app expects. Must be one of {param_value_types.__values__}.")  # pytype: disable=attribute-error
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/typing.py", line 760, in __getattr__
    raise AttributeError(attr)
AttributeError: __values__
```

**Comment by keighrim** [comment]
closed via #74.
